src/main/java/io/kestra/plugin/gcp/dataform/package-info.java

```diff
@@ -0,0 +1,9 @@
+@PluginSubGroup(
```

Comment:
Missing icon if we have a sub group

<img width="512" height="512" alt="Image" src="https://github.com/user-attachments/assets/8684df21-891d-4760-8b14-413c89b0462f" />)